images/virtualization-artifact/pkg/controller/common/util.go

```diff
@@ -550,3 +548,25 @@ func SetRecommendedLabels(obj metav1.Object, installerLabels map[string]string,
 
 	obj.SetLabels(mergedLabels)
 }
+
+const ipPrefix = "ip-"
+
+// IpToLeaseName generate the Virtual Machine IP Address Lease's name from the ip address
+func IpToLeaseName(ip string) string {
+	addr := net.ParseIP(ip)
+	if addr.To4() != nil {
+		// IPv4 address
+		return ipPrefix + strings.ReplaceAll(addr.String(), ".", "-")
+	}
+
+	return ""
+}
+
+// LeaseNameToIP generate the ip address from the Virtual Machine IP Address Lease's name
+func LeaseNameToIP(leaseName string) string {
+	if strings.HasPrefix(leaseName, ipPrefix) && len(leaseName) > len(ipPrefix) {
+		return strings.ReplaceAll(leaseName[len(ipPrefix):], "-", ".")
+	}
+
+	return ""
+}
```

Comment:
Please, move it to a new file here in common package, e.g. "ip.go", or into the ipam package.